build.gradle

```diff
@@ -7,164 +7,226 @@ version = '0.1.0'
+                Recipe is a json file that describes the data or models that we want to get out of the system, without 
+                describing how to generate the data. The recipe language is expressed in JSON format. 
+
+-Ps             -Ps
+                Optional property indicating that the recipe contains a recipe string and not the path.
+                Used primaryly when the command is run from insid code not on the commandline.
+
+-Poutput        -Poutput=path/to/output/file
+                Mandatory absolute path to the output file with the model results.
+
+-Pcorrelation   -Pcorrelation=path/to/correlation/analisys/output/file
+                Optional absolute path to the output file with the correlation analysis' results.
+                Calculates correlation between all input fields and output the Pearson correlation coefficient,
+                the pValue and the standard error to a JSON file. 
+                Note: only supports the use of GeoJson as the intermediate data export format.
+
+-Pforce         -Pforce=className:datasourceId,...
+                Optional list of classes to reimport.
+                It will re-download from the source and update the local file.
+
+-Pclear         -Pclear=true/false
+                Optional boolean triggering the database cache to be cleaned so the data will be persisted again.
+
+Examples:
+
+gradle runExport -Precipe='src/main/resources/executions/examples/greenspace-hertfordshire.json' -Poutput='output.json' -Pforce='uk.org.tombolo.importer.osm.OSMImporter' -Pclear=true
+
+gradle runExport -Precipe='src/main/resources/executions/examples/london-cycle-traffic-air-quality-lsoa.json' -Poutput='output.json'
+
+gradle runExport -Precipe='src/main/resources/executions/examples/london-cycle-traffic-air-quality-lsoa.json' -Poutput='output.json' -Pcorrelation='correlation_output.json'
+
+
+"""
+}
+
+task runExport(dependsOn: ['classes', runExportHelp], type: JavaExec) {
+    main = 'uk.org.tombolo.DataExportRunner'
+    classpath = sourceSets.main.runtimeClasspath
+    jvmArgs '-disableassertions'
+
+    doFirst {
+        checkEnv()
+        systemProperty("databaseURI", databaseURI)
+        systemProperty("databaseUsername", databaseUsername)
+        systemProperty("databasePassword", databasePassword)
+        systemProperty("environment", "export")
+
+        def argumentsSet = project.hasProperty('recipe') && project.hasProperty('output');
+        if (!argumentsSet) {
+            print(runExportHelp.info)
+            throw new StopExecutionException()
+        }
+        if (!project.hasProperty('clear')) { ext.clear = false }
+        if (!project.hasProperty('force')) { ext.force = "None" }
+        if (!project.hasProperty('correlation')) { ext.correlation = "None" }
+        ext.s = false
+        if (project.hasProperty('s')) { ext.s = true }
+
+        args(s, recipe, output, correlation, force, clear)
+    }
 }
```

Comment:
Noticed two things here:
1) It only works if user while running the task ```runExport``` forgets to put some parameters, not with any other tasks
2) If user want to see from the command line ```how to run recipe or use digital connector``` they can't print help, i think it would be useful if user could print help, so that they know how to use Digital connector
3) If error happens inside the digital connector due to incorrect path, as user may be dont know how to provide path, help page didn't get print either
4) i think it would be good if we have a format like ```gradle runExport help``` for every task

Reply:
1, 2, 4 - In this PR I only did the man page for runExport, and decided to leave `runCatalogue` and `test` because I thought they are less likely to need a help page given their use.
It works also if the user runs `gradle runExport` or `gradle runExport -Phelp`. I prefer having the help page shown when I run the command without arguments. 
3 - Do you mean that the help page should be printed at every error related to user command line input? Or always when there is an error? The first one might be a little bit more tricky to do if wanting to use the info in the gradle  help task. I'll have a look at it.

Reply:
no only when there is an error related to user input in running the command

Reply:
I think this  could be an addon to current functionality